data/news/alameda.json

```diff
@@ -24,8 +24,9 @@
     {
       "id": "https://covid-19.acgov.org/covid19-assets/docs/press/press-release-2021.05.13.pdf",
       "url": "https://covid-19.acgov.org/covid19-assets/docs/press/press-release-2021.05.13.pdf",
-      "title": "Safe, Effective COVID-19 Vaccine Approved for 12-15 Years of Age; Family Days and Drop-In Access for Priority Neighborhoods",
-      "date_published": "2021-05-13T00:00:00-07:00"
+      "title": "Safe, Effective COVID-19 Vaccine Approved for 12-15 Years of Age; Family Days and Drop-In Access for Priority Neighborhood",
+      "date_published": "2021-05-13T00:00:00-07:00",
+      "summary": "English |"
```

Comment:
This will be fixed in https://github.com/sfbrigade/data-covid19-sfbayarea/pull/210